### DISCUSSION TITLE
Index the website field in profiles?

### DISCUSSION THREAD

**Discussion by snarfed** [community/community#38517] [Other Feature Feedback, Questions, & Ideas]
Hi all! I recently noticed that the website field in user profiles isn't indexed for search. For example, https://github.com/snarfed has website https://snarfed.org/ , but [user searches for snarfed.org](https://github.com/search?q=snarfed.org&type=users) [and https://snarfed.org/](https://github.com/search?q=https://snarfed.org/&type=users) both return no results.

Thanks in advance for considering this!

**Comment by ossanna16** [comment]
Hi there @snarfed and welcome to our community! Thank you for the feedback!

**Comment by jfullstackdev** [comment]
it's 

<https://github.com/search?q=snarfed&type=users>

![image](https://user-images.githubusercontent.com/47092464/202485108-fbb93251-eb55-428c-b3ff-f65ba175e4db.png)

**Comment by snarfed** [comment]
@jdevstatic the request is to index web site, not username.

**Reply by jfullstackdev** [reply]
yup, I see that, GitHub is indexing the users, not like the usual indexing of search engines because it's primary concern is for GitHub users, not any other external site

in the past, I use Google Search Console and Google Analytics for the traffic of my site, even if I'm hosting it using GitHub Pages

but I'm not very sure if it's really indexing external sites because that will qualify GitHub Search as a search engine

**Comment by snarfed** [comment]
I'm not looking for it to index my web site's contents, just the literal strings `https://snarfed.org/` and/or `snarfed.org`, like in the search links in my original description above.
